tfhe/src/shortint/parameters/parameters_wopbs.rs

```diff
@@ -7,9 +7,33 @@ pub use crate::core_crypto::commons::parameters::{
 use crate::shortint::parameters::{
     CarryModulus, CiphertextModulus, EncryptionKeyChoice, MessageModulus,
 };
-use crate::shortint::Parameters;
+use serde::{Deserialize, Serialize};
 
-pub const ALL_PARAMETER_VEC_WOPBS_NORM2: [Parameters; 31] = [
+/// A structure defining the set of cryptographic parameters for homomorphic integer circuit
+/// evaluation.
```

Comment:
Maybe this doctring needs a little update, mainly to emphasize that these are meant for WopPBS

Reply:
definitely

Reply:
done